fusor-ember-cli/app/templates/storage.hbs

```diff
@@ -19,8 +19,8 @@
       </span>
     {{/radio-button}}
     &nbsp;&nbsp;&nbsp;&nbsp;
-    {{#radio-button value="Gluster" groupValue=model.rhev_storage_type id='gluster' disabled=true}}
-      <span class="disabled">
+    {{#radio-button value="glusterfs" groupValue=model.rhev_storage_type id='gluster' disabled=deploymentController.isStarted}}
```

Comment:
Not your doing, but, does the "isStarted" for the radio button for "Local" up above work?  if not I'm guessing that it should be an alias you could reuse.  I was expecting to see the following in storage.js:

```javascript
isStarted: Ember.computed.alias('deploymentController.isStarted'),
```